VERSIONING.md

```diff
@@ -0,0 +1,63 @@
+# Versioning strategy
+
+This repository follows a [CalVer](https://calver.org/)-type approach to versioning, in contrast to the majority of
+other PyGranta and PyAnsys projects which follow [SemVer](https://semver.org/). 
+
+This package is versioned according to the following scheme: `YYYY.[1,2].MINOR`
+
+The reasons for this are described below.
+
+# Advantages of this approach
+
+This repository only contains code generated from OpenAPI JSON definition files, which themselves are versioned 
+with Granta MI. Granta MI follows a CalVer-type approach of `YYYY.[1,2]`, with two versions released per year.
+
+These OpenAPI JSON definition files are _backwards-compatible_, but not _forwards-compatible_. In other words,
+the API definition from an older version of Granta MI is compatible with a newer version, but not the other way
```

Comment:
Reading this, I found myself asking 'newer version of what?' This might be obvious to the intended audience, but it might be helpful to clarify 'a newer version of the OpenAPI JSON definition files'